plugins/filter_kubernetes/kube_meta.c

```diff
@@ -306,6 +306,12 @@ static void cb_results(const char *name, const char *value,
         meta->container_hash_len = vlen;
         meta->fields++;
     }
+    else if (meta->container_image == NULL &&
```

Comment:
You don't need this block. This part is only (if I am not mistaken) for meta information which could be extracted from Tag/Filename. Like: app-7957d44d8b-8sfjr_default_app-da54cb58f3cb4e0e68047a341ee0a63498c19a675cdfc1203d00d64525833c44.log

pod_name = app-7957d44d8b-8sfjr
namespace_name = default
container_name = app
docker_id = da54cb58f3cb4e0e68047a341ee0a63498c19a675cdfc1203d00d64525833c44

Reply:
yor're right, cb_results is called from flb_regex_parse - will remove it.